### PR TITLE
feat: OS pages content & hero landing page layout 

### PR DESCRIPTION
**Pull request by vinckr**
https://github.com/ory/web/pull/249 redone on web branch

The change from #248 had to be included for styling.
If that PR needs changes I can change it here accordingly, but the layout was important for both.

Read more about my reasons for these changes here: ory-corp/product#151 (comment)

### CODE REVIEW

File: src/components/layouts/hero/hero.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ interface PropTypes {
 const Hero = ({ title, description, image, buttons }: PropTypes) => (
   <div className={cn(hero)}>
     <Container fluid={true} alignItems={'center'}>
-      <Grid lg={4} md={3} sm={12} xs={12} className={cn(pb32)}>
+      <Grid lg={6} md={7} sm={12} xs={12} className={cn(pb32)}>
```
Comment (jfcurran) [inline src/components/layouts/hero/hero.tsx:23]:
```suggestion
      <Grid lg={5} md={7} sm={12} xs={12} className={cn(pb32)}>
```
I see why this is intended to fill half of the available space, but I disagree with doing so, as this decreases space around the text too much. Leaving it at 5 gives the content room to breathe

Comment (jfcurran) [reply src/components/layouts/hero/hero.tsx:23]:
```suggestion
      <Grid lg={6} md={6} sm={12} xs={12} className={cn(pb32)}>
```

> I see why this is intended to fill half of the available space, but I disagree with doing so, as this decreases space around the text too much. Leaving it at 5 gives the content room to breathe

the same reasoning applies here, although there is less real estate to begin with. I believe a 6/6 split works best here, and images are still displayed at a reasonable size for big tablets / small laptops

File: src/components/layouts/hero/hero.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ const Hero = ({ title, description, image, buttons }: PropTypes) => (
           </MoleculeTextInteraction>
         </ContentText>
       </Grid>
-      <Grid lg={6} md={8} sm={12} xs={12}>
+      <Grid lg={6} md={5} sm={12} xs={12}>
```
Comment (jfcurran) [inline src/components/layouts/hero/hero.tsx:34]:
```suggestion
      <Grid lg={6} md={6} sm={12} xs={12}>
```
see my previous comment for reasoning behind this suggestion

File: src/pages/hydra.tsx [hunk 3]
```diff
@@ -56,6 +56,9 @@ const IndexPage = () => {
   const ArrowRight = (
     <IconWrapper color={'themed-primary'} icon={'ArrowRight'} size={'16'} />
   )
+  const CheckCircle = (
+    <IconWrapper color={'base-white'} icon={'CheckCircle'} size={'24'} />
```
Comment (jfcurran) [inline src/pages/hydra.tsx:60]: 😎 👍

File: src/pages/keto.tsx [hunk 4]
```diff
@@ -122,16 +133,16 @@ const IndexPage = () => {
...
         description={
           <>
-            Ory Keto is a global and consistent permission & authorization
-            server with an easy and granular permission language and sub
-            10-millisecond latency. It is based on Google Zanzibar, written in
-            Go, and ships gRPC and REST APIs.
+            Ory Keto is an Open Source global and consistent permission &
+            authorization server with an easy and granular permission language
+            and low latency. It is based on Google Zanzibar, written in Go, and
+            ships gRPC and REST APIs.
```
Comment (jfcurran) [inline src/pages/keto.tsx:139]: great

**Review by jfcurran — COMMENTED** [review]
Hey @vinckr great stuff, please see my comments regarding the use of the grid - I think its important we remain consistent regarding how images and text are displayed. My presumption is this has to do with how headlines look: although your suggested change works great for index.tsx for example, I think its lacking on hydra.tsx or keto.tsx

File: src/components/layouts/hero/hero.tsx (hunk 2 above)
Comment (vinckr) [reply src/components/layouts/hero/hero.tsx:34]:
```suggestion
      <Grid lg={7} md={6} sm={12} xs={12}>
```
lg has to be 7 right?

File: src/components/layouts/hero/hero.tsx [hunk 5]
```diff
@@ -31,7 +31,7 @@ const Hero = ({ title, description, image, buttons }: PropTypes) => (
           </MoleculeTextInteraction>
         </ContentText>
       </Grid>
-      <Grid lg={6} md={8} sm={12} xs={12}>
+      <Grid lg={7} md={6} sm={12} xs={12}>
```
Comment (jfcurran) [inline src/components/layouts/hero/hero.tsx:34]:
```suggestion
      <Grid lg={6} md={6} sm={12} xs={12}>
```
the grid props dont need to add up to `{12}`, but they should not exceed `{12}`

**Review by jfcurran — CHANGES_REQUESTED** [review]

**Comment by vinckr** [comment]
Should I make a separate PR for the hero text changes? I could just add the commit in here.

**Comment by jfcurran** [comment]
lets merge this, and discuss the content changes in a different PR. I think it's good to add reviewers (if needed) in a specific PR :) 👍

**Pull request merged**